src/Layout/redist/targets/OverlaySdkOnLKG.targets

```diff
@@ -93,6 +96,9 @@
     <Exec Condition="Exists('$(RedistLayoutPath)TestRollback.json')"
           Command="$(RedistLayoutPath)dotnet workload update --from-rollback-file $(RedistLayoutPath)TestRollback.json"
           EnvironmentVariables="DOTNET_CLI_HOME=$(ArtifactsTmpDir)"/>
+
+    <OverrideWasmRuntimePackVersions Properties="@(WasmRuntimePackVersion)" 
+      WorkloadManifestPath="$(RedistLayoutPath)\sdk-manifests\$(MonoWorkloadFeatureBand)\microsoft.net.workload.mono.toolchain.current\WorkloadManifest.targets" />
```

Comment:
This will need to be changed when we switch to side-by-side manifests.

Reply:
I hope to have a better solution sooner...